### ISSUE TITLE
[Jira] Missing `title` on Open Issues command

### ISSUE THREAD

**Issue by por** [raycast/extensions#6011]
### Extension

https://www.raycast.com/raycast/jira

#### Error:
```
Error: Rendering failed:
Missing required component property "title"
```

### Steps To Reproduce

1. In Production
2. Logged in as (me)
3. Run "Open Issues" or "Watched Issues" command
4. All good
5. Create an issue via the "Create Issue" command
6. Run "Open Issues" or "Watched Issues" command
7. Receive an error

![Screenshot 2023-04-18 at 14 22 08](https://user-images.githubusercontent.com/67254/232775871-0620801d-7a9b-4402-93d4-a468e9719ee5.png)

### Current Behaviour

It seems like the API returns an issue with a missing title? But when I open the Jira list in the browser there are no issues without a title. I can’t debug this unfortunately without knowing which issue it involves.

### Expected Behaviour

This command used to work. I’m expecting to see a list of issues assigned to me or ones I am watching. This used to work until a couple of days ago. It seems to break after running the "Create Issue" command. If the API returns an issue without a `title`, maybe it should be filtered out before trying to render the list.

**Comment by thomaslombart** [comment]
Unfortunately, I still can't reproduce. I'm not using Jira so I'm using it on a test instance which is far from real-world conditions. I've [pushed a fix one hour ago](https://github.com/raycast/extensions/commit/af662c71347a11ce8993de3d215d2017747cd247) that may fix the issue (don't forget to run `Check For Extension Updates`). Do you still have the error?

If you still have it, it'd be super useful if you could fork the repo, log your issues in the `StatusIssueList` component and let me know what issues you're getting. This is typically a case where I'm debugging this issue blindly 😬

**Comment by thomaslombart** [comment]
Thanks for your time 🙏

**Comment by por** [comment]
@thomaslombart So I forked and could not reproduce the issue anymore with the fork. Still happening on the store version. Does that give you any clues? I tried multiple times. Maybe someone else can verify? I’m running out of issues to create in JIRA to debug this :)

Is there perhaps any other way to debug this with the store-installed extension?

**Comment by thomaslombart** [comment]
Thanks for trying it. This issue is hard to debug indeed. It seems to happen randomly and I can't reproduce either locally or in the store. So, I'll check with the team if something's happening internally.

**Comment by dmytroyaremenkoleanix** [comment]
The same after creating Jira issues via Raycast

**Comment by chrismetcalf** [comment]
👋 Just checked for updates and I'm still seeing the same too. I'm able to view roughly the same list of tickets through the "My Filters" view with an appropriate filter.

**Comment by thomaslombart** [comment]
Hey @por, I prepared a fix over [here](https://github.com/raycast/extensions/pull/6291) that may fix your issue. It seems related to issues having statuses with no categories. Could you check out the branch and let me know if you still run into the issue?

**Comment by por** [comment]
@thomaslombart Thanks! I haven’t been able to reproduce the issue myself for the last few weeks, so can’t verify if this fix works, but looks good 👍

**Comment by chuxubank** [comment]
Hi @thomaslombart , I verified your branch by using `Import Extension` and it works well!
Thanks for your fix and hoping it get merged soon 😆

**Comment by thomaslombart** [comment]
Awesome. I'll merge that today.

**Comment by thomaslombart** [comment]
You shouldn't have to uninstall the extension, though. Updates land a bit later for extensions, so you have to run `Check for Extension Updates` sometimes.

**Comment by colinfitzpatrick** [comment]
This issue is still occurring for me.  I removed the old extension and installed the latest today (May 11th, 2023).

Logging out and logging back in again doesn't resolve the issue.  I need to uninstall and reinstall the extension to correct it.

**Comment by colinfitzpatrick** [comment]
Thank you.

If it's easier, I recorded by screen (in case I'm doing something incorrectly)
https://www.icloud.com/iclouddrive/0782Y19_HTDhnvNzGAANF-HKQ#My_Movie_2

1. Install Jira Extension
2. Go to Open Issues (I log in)
3. I can see Open issues
4. Create a new issue (this works)
5. Go to Open Issues, and I get the Missing title error.

**Comment by chuxubank** [comment]
Unfortunately, I meet same error yesterday.
I also see "Missing title error" after create a issue using Raycast.
A fresh install can solve this issue (temporarily, before create issue using Raycast).

**Comment by thomaslombart** [comment]
Thanks for the bug report, does that happen only after creating a new issue or does it happen also when opening the `Open Issues` command?

**Comment by colinfitzpatrick** [comment]
It works perfectly until I create a new issue with the extension.

**Comment by thomaslombart** [comment]
@colinfitzpatrick Do you have any additional custom fields when creating your issue?

**Comment by colinfitzpatrick** [comment]
The project has some custom fields (see attached screenshot).

If it helps, I've also noted that I don't need to create a new issue for the problem to occur, viewing "create issue" is enough to cause the problem.

![Screenshot 2023-05-15 at 10 55 15](https://github.com/raycast/extensions/assets/21079269/5dac2093-0fa6-4e00-bf64-c103d102b282)

**Comment by thomaslombart** [comment]
Thanks for the additional details. I'm reopening this issue then.

**Comment by colinfitzpatrick** [comment]
I just tried it now, and it fixed the issue for me.

Thank you.
